code/modules/mob/living/silicon/robot/robot_vr.dm

```diff
@@ -17,24 +18,22 @@
 
 /mob/living/silicon/robot/updateicon()
 	..()
-	if(icon == 'icons/mob/widerobot_vr.dmi' || 'icons/mob/64x64robot_vr.dmi')
-		if(stat == CONSCIOUS)
-			if(sleeper_g == 1)
-				overlays += "[module_sprites[icontype]]-sleeper_g"
-			if(sleeper_r == 1)
-				overlays += "[module_sprites[icontype]]-sleeper_r"
-			if(istype(module_active,/obj/item/weapon/gun/energy/laser/mounted))
-				overlays += "laser"
-			if(istype(module_active,/obj/item/weapon/gun/energy/taser/mounted/cyborg))
-				overlays += "taser"
-			if(resting)
-				overlays.Cut() // Hide that gut for it has no ground sprite yo.
-				icon_state = "[module_sprites[icontype]]-rest"
-			else
-				icon_state = module_sprites[icontype]
-		else if(stat == DEAD)
-			icon_state = "[module_sprites[icontype]]-wreck"
-			overlays += "wreck-overlay"
+	if(dogborg == TRUE && stat == CONSCIOUS && sleeper_g == TRUE)
```

Comment:
Why this change? It was actually much more efficient, CPU-time-wise, before.
Previously it'd only check if they were conscious once and if they were a dogborg (based on the poor icon test, glad you fixed that) once.

Now it checks both 10 times for some reason?